components/chat_widget/src/components/ocs-chat/ocs-chat.css

```
@@ -638,6 +638,22 @@
     @apply flex gap-[8px];
   }
 
+  .consent-panel {
+    @apply flex flex-col gap-3;
+    padding-bottom: 1em;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)